plugins/woocommerce/client/blocks/assets/js/blocks/product-collection/collections/cross-sells.tsx

```diff
@@ -27,13 +31,15 @@ const collection = {
 	usesReference: [ 'product', 'cart', 'order' ],
 };
 
-const attributes = {
+export const attributes = {
+	...DEFAULT_ATTRIBUTES,
 	displayLayout: {
 		type: 'flex',
 		columns: 4,
 		shrinkColumns: true,
 	},
 	query: {
+		...DEFAULT_QUERY,
 		perPage: 8,
```

Comment:
I saw @straku's comment and agree! I think we should adjust it to what was there in old block: 3 products and 3 columns.

You can either do it globally here (which I think is ok anyway, 8 products in 2 rows is a lot!) or overwrite the attributes in each place you define new block.

If changed globally here, please change it in upsells as well.

Reply:
I won't change globally, I only changed the instances of cross-sells added to the cart. I did this because it can appear on the single product page as well.

I didn't add a cart instance for upsells, because that's not in the cart template, nor the cart editor.